### ISSUE TITLE
doi does not show up in Ocellus

### ISSUE THREAD

**Issue by myrmoteras** [punkish/ocellus#39]
@punkish I am trying to look up the latest figures in TB, 20230513 https://tb.plazi.org/GgServer/html/039E87EDFFF2FFDDFF67FD94FB28CC28 with `10.11646/zootaxa.5284.3.7`

This does not provide a result `sorry, no images found where articleDOI is 10.11646/zootaxa.5284.3.7`

if I use the latest EJT, then it shows up https://ocellus.info/?page=1&size=30&resource=images&articleDOI=10.5852%2Fejt.2023.870.2113 

Could this be because it takes a certain amount until new data is in Zenodeo/Ocellus? If so, what is the update frequency?

**Comment by punkish** [comment]
yes, it could be… in theory, Zenodeo updates every night, but right now there is a glitch in the auto update mechanism that I am trying to track down. I will let you know soon as that is working reliably

**Comment by myrmoteras** [comment]
there must be something different than latency. till does not work https://ocellus.info/?page=1&size=30&resource=images&articleDOI=10.11646%2Fzootaxa.5284.3.7

**Comment by punkish** [comment]
it is not latency… as I said in my response, "there is a glitch in the auto update mechanism that I am trying to track down". I think I have found it, but am still working on a reliable fix. Will let you know soon as it is working

**Comment by punkish** [comment]
I updated the db manually and your images are now showing up. I think I know why the autoupdate is not working, but it has to be tested for reliability. So, there may be such glitches again until I've got it down working well. For now, your images show up. Try that link again https://ocellus.info/?page=1&size=30&resource=images&articleDOI=10.11646%2Fzootaxa.5284.3.7
